### PR TITLE
Update some functions in examples

### PR DESCRIPTION
**Pull request by cesarcosta99**
According to issue #42.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -627,9 +627,7 @@ p.then((val) => console.log("Promise Resolved", val),
 Returning a value from `then` callbacks will pass the value to the next `then` callback.
 
 ```javascript
-var hello = new Promise(function(resolve, reject) {  
-    resolve("Hello");
-});
+var hello = new Promise((resolve, reject) => resolve("Hello"));
```
Comment (metagrover) [inline README.md]: Hi @cesarcosta99, you are returning resolve("Hello") here, you should instead write it as:

`new Promise((resolve, reject) => { resolve("Hello") });`

Similarly at line 641 and 643

**Review by metagrover — CHANGES_REQUESTED** [review]
Should not return a resolve().

**Comment by GoToLoop** [comment]
`new Promise((resolve, reject) => { resolve("Hello") });`

Given it's only 1 statement, there's no need to wrap it w/ `{}`:    ;-D
`new Promise((resolve, reject) => resolve('Hello'));`

Unless of course you really want/need  that lambda expression to return `undefined` instead.   O_o

**Comment by metagrover** [comment]
I've seen both the formats used in various codebases, but what's the point of returning `resolve('Hello')`?

Here, we are executing a statement, not returning anything, so it makes sense to wrap it around `{ }`

**Comment by GoToLoop** [comment]
The less boilerplate the better. If there's no problem returning **resolve()**, the lambda expression is cleaner w/o the extra `{}`!

**Comment by cesarcosta99** [comment]
@GoToLoop said what I was thinking; results in the same. But that's alright, I going to change to add the curly brackets.

**Review by metagrover — APPROVED** [review]

**Comment by metagrover** [comment]
Sorry guys, I got busy yesterday. I also agree with @GoToLoop, absence of `{}` makes the code  a little prettier. But, now that the changes have been submitted, I accepted the PR.

Cheers.

**Pull request merged**